rdt/transformers/datetime.py

```diff
@@ -168,11 +162,9 @@ class OptimizedTimestampEncoder(UnixTimestampEncoder):
             Indicate what to replace the null values with. If the strings ``'mean'`` or ``'mode'``
             are given, replace them with the corresponding aggregation.
             Defaults to ``mean``.
-        model_missing_values (bool):
```

Comment:
Shouldn't we remove the docstrings when the parameter is removed? Keeping it there with the `DEPRECATED` message makes it clear what's going on, while removing it just makes it seem we forgot the docstring of the extra parameter.

Reply:
I agree that we shouldn't get rid of the docstring